src/rpc/client.cpp

```diff
@@ -108,11 +108,13 @@ static const CRPCConvertParam vRPCConvertParams[] =
     { "combinerawtransaction", 0, "txs" },
     { "fundrawtransaction", 1, "options" },
     { "fundrawtransaction", 2, "iswitness" },
+    { "fundrawtransaction", 3, "solving_data" },
```

Comment:
I suppose you ought to add these to `send` from #16378 as well

Reply:
I've added `solving_data` to `send` via `options`.